ci/cloudbuild/builds/checkers.sh

```diff
@@ -97,15 +114,12 @@ time {
 # includes.
 printf "%-30s" "Running Abseil header fixes:" >&2
 time {
+  expressions=("-e" "'s;#include \"absl/strings/str_\(cat\|replace\|join\).h\";#include \"google/cloud/internal/absl_str_\1_quiet.h\";'")
```

Comment:
Nit: The double quoting is a pain.

Reply:
Agreed. Not sure how to make that better. Open to suggestions.